packages/spectral/package.json

```diff
@@ -23,6 +26,9 @@
   "scripts": {
     "clean": "rm -rf dist",
     "build": "yarn run format && yarn run lint && yarn run clean && tsc",
+    "build:templates": "copyfiles --error -u 1 src/**/templates/**/* dist",
+    "cli:permission": "chmod +x dist/generators/**/cli.js",
```

Comment:
When running the command `npm run build` to generate the Spectral package, the file permissions are being stripped as the files are added to the `dist` directory. We need to ensure that the command-line interface (CLI) executables in the generators directory have the correct executable permissions. While the `chmod` command works in Linux environments, we're not sure if it's compatible with our Continuous Integration (CI) environments. Therefore, we'll need to find a more suitable command to set the necessary permissions for those CLI executables.